src/main/java/com/starbucks/repository/QueryRepositary.java

```diff
@@ -0,0 +1,18 @@
+package com.starbucks.repository;
+
+import com.starbucks.model.Order;
+import com.starbucks.model.Status;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+import java.util.List;
+import java.util.UUID;
+
+public interface QueryRepositary extends JpaRepository<Order, UUID> {
+
+    @Query("SELECT o from Order o where o.status='READY'")
+    List<Order> findAllActiveUser();
```

Comment:
You can try to do it with something like fineAllByStatus and send the status as a parameter.